homeassistant/components/tuya/light.py

```diff
@@ -18,6 +18,9 @@
 from .const import DOMAIN, TUYA_DATA, TUYA_DISCOVERY_NEW
 
 PARALLEL_UPDATES = 0
+NORMAL_LIGHT_RANGE = (0, 255)
+TUYA_LIGHT_TURN_ON_RANGE = (28, 255)
+TUYA_LIGHT_RANGE = (10.3, 1000)
 
```

Comment:
Sorry for this, my mistake, but I did some addidtional test and probably right value (at least for me) are:

```
NORMAL_LIGHT_RANGE = (1, 255)
TUYA_LIGHT_TURN_ON_RANGE = (28.3, 255)
TUYA_LIGHT_RANGE = (10, 1000)
```

This work perfect for the only bulb that I have, that means that at minimum value the light do not turn off and brightness percentage match 100% with Tuya app.

Reply:
```suggestion
NORMAL_LIGHT_RANGE = (1, 255)
TUYA_LIGHT_TURN_ON_RANGE = (28.3, 255)
TUYA_LIGHT_RANGE = (10, 1000)
```